hail/python/hail/backend/service_backend.py

```diff
@@ -359,6 +323,10 @@ def debug_info(self) -> Dict[str, Any]:
     def fs(self) -> FS:
         return self._sync_fs
 
+    @property
+    def jar_spec(self) -> dict:
+        return {'type': 'git_revision', 'value': revision()}
```

Comment:
To be clear, we need to `make upload-for-qob` on our branches to use qob?

Reply:
Yes we do